lib/form/html.js

```diff
@@ -90,6 +90,7 @@ const addRequired = (html, input, index) => {
 
     if (input.required === 'save' || inp.required === 'save' || input.required === 'publish' || inp.required === 'publish') {
       let level = '';
+      const required = input.required && inp.required;
```

Comment:
@scottnath - updated and tested locally but got the following error:

```
94:13  error  'level' is already defined  no-redeclare
```

Test passed after that.

So I removed this:
```
let level = '';
```